Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -103,6 +103,15 @@ def _get_test_data_json_string(self, temp_result_file):
             del temp_result_json['debugOutput']
         return json.dumps(temp_result_json)
 
+    # urllib.request.urlopen always rises an exception when the http return code is not 200
+    # so this wraps the call to return the HTTPError object instead of rising the exception.
+    # The HTTPError object can can be treated later as a http.client.HTTPResponse object
```

Comment:
No need to dance can-can here, one “can” 🐶 is enough:

 ```suggestion
    # The HTTPError object can be treated later as a http.client.HTTPResponse object
```